tests/v2/validation/charts/webhook.go

```diff
@@ -0,0 +1,28 @@
+package charts
+
+import (
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/kubeapi/webhook"
+)
+
+const (
+	resourceName           = "rancher.cattle.io"
+	restrictedAdmin        = "restricted-admin"
+	admin                  = "admin"
+)
+
+
+func getWebhook(client *rancher.Client, clusterID, resourceName string) ([]string, error) {
```

Comment:
Small nit, maybe changing this name to `getWebhookNames`?

Reply:
Resolved in the other PR. Once that is merged will rebase this PR again with those changes.